packages/common/src/store/reachability/reducer.ts

```diff
@@ -6,7 +6,7 @@ import { ReachabilityState } from './types'
 type ReachabilityActions = ActionType<typeof actions>
 
 const initialState = {
-  networkReachable: false
+  networkReachable: true
```

Comment:
cc @amendelsohn You think this is safe?

Reply:
I tested:
- going into offline mode with app open and clicking around
- going into app from already offline

Both seem to work well. Lmk if I should test more!

Reply:
Hmm so iirc the reason for this was to keep us from making and failing a bunch of requests that we assume would succeed.
The tricky case is:
1. start the app offline
2. wait for the initial feed lineup fetch to fail
3. go back online
4. make sure the lineup request gets sent again on reconnect

Reply:
1-4 seems to work fine for me with the changes here! 🤔 

https://user-images.githubusercontent.com/2731362/203259305-668066e2-ca1a-4d86-86df-b804e7e095b8.MP4

I'm going to merge this so it gets build to staging apps, but will follow on

Reply:
I remember it mattered whether I actually waited for the previous request to fire and fail while offline. Basically, I think when we do it quickly, either the saga or the request can still be queued up and just proceed when connected. It looks like you turned the network back on quickly after opening. We should make sure it still works after waiting ~20s or until the `feedFetchMetadatas` call fails (if we're still making it at all on startup).

Reply:
Pretty sure this happened because if we default to `reachable = true`, we initialize the backend and apiClient on startup before we get a chance to switch reachable to false, and also may briefly render the Feed screen content before we switch to the offline indicator. This may not be a problem anymore, but it's worth testing.